icons/alby.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Alby</title><path d="M12.1 9.3c-3.8 0-7 2.4-8 5.5-.4 1 .3 2.1 1.3 2.6 2.1.8 4.3 1.3 6.8 1.3 2.4 0 4.7-.5 6.8-1.3 1-.4 1.6-1.6 1.3-2.6a8.78 8.78 0 0 0-8.2-5.5zM9.2 16c-1 0-2-.7-2-1.6s.9-1.6 2-1.6 2 .7 2 1.6c-.1.8-1 1.6-2 1.6zm5.6 0c-1 0-2-.7-2-1.6s.9-1.6 2-1.6 2 .7 2 1.6c0 .8-.9 1.6-2 1.6zm4.7-8.2.9-.9c.4.3.7.3 1.2.3 1.3 0 2.4-1 2.4-2.4s-1-2.4-2.4-2.4-2.4 1-2.4 2.4c0 .4.1.9.4 1.2l-.9.9a10.3 10.3 0 0 0-13.4 0l-1-.9c.3-.3.4-.8.4-1.2 0-1.3-1-2.4-2.4-2.4S0 3.5 0 4.8s1 2.4 2.4 2.4c.4 0 .8-.1 1.2-.3l.9.9c-2 2.1-3.4 4.7-3.8 7.6-.3 1.7.7 3.3 2.1 3.9 2.9 1.3 6 2.1 9.3 2.1s6.4-.8 9.3-2.1c1.4-.7 2.4-2.4 2.1-3.9-.5-3-2-5.6-4-7.6zm1.4 10.6c-2.6 1.2-5.6 2-8.8 2s-6.2-.7-8.8-2c-1-.5-1.6-1.6-1.4-2.8 1-5.8 5.2-10 10.2-10s9.2 4.2 10.2 10c.1 1.1-.4 2.3-1.4 2.8z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Alby</title><path d="M.686 15.453c.536-2.996 1.908-5.66 3.83-7.604l-.98-.98a2.362 2.362 0 1 1 .844-.822L5.384 7.05c1.884-1.57 4.19-2.5 6.7-2.5 2.478 0 4.756.908 6.629 2.446l.922-.922a2.362 2.362 0 1 1 .854.812l-.904.902c1.96 1.958 3.358 4.66 3.892 7.704h.002a3.7 3.7 0 0 1-2.106 3.99 21.73 21.73 0 0 1-9.255 2.06 21.62 21.62 0 0 1-9.34-2.1l.508-1.068a20.476 20.476 0 0 0 8.832 1.99 20.5 20.5 0 0 0 8.753-1.95c1.05-.49 1.64-1.596 1.448-2.71-1.016-5.774-5.279-9.976-10.235-9.976S2.882 9.907 1.852 15.667a2.524 2.524 0 0 0 1.434 2.708l-.508 1.062a3.708 3.708 0 0 1-2.092-3.982m18.17 1.974a17.93 17.93 0 0 1-6.778 1.32 17.984 17.984 0 0 1-6.776-1.32c-1.024-.418-1.63-1.506-1.276-2.554 1.096-3.232 4.288-5.568 8.052-5.568s6.959 2.336 8.053 5.568c.36 1.048-.252 2.136-1.274 2.554zm-7.724-3.082c0-.868-.88-1.57-1.97-1.57-1.086 0-1.968.702-1.968 1.57 0 .868.88 1.57 1.968 1.57s1.97-.702 1.97-1.57zm5.632 0c0-.868-.88-1.57-1.97-1.57s-1.968.702-1.968 1.57c0 .868.88 1.57 1.968 1.57s1.97-.702 1.97-1.57z"/></svg>
```

Suggestion to fix the linter error

Reply:
In the preview image the logo looks a bit jagged. I created this preview using the changes I suggested.
![alby](https://github.com/simple-icons/simple-icons/assets/35000807/514edd01-5fcb-4380-a62c-606c3f80d3cd)